### PR TITLE
French translation of 4.6.3 announcement

### PR DESCRIPTION
**Pull request by dje29**
This should be good enough. @Ghalr, @pypingou and @alexandrebelloni if you want to have a look, any improvement proposal is welcome :)

### CODE REVIEW

**Comment by Ghalr** [comment]
I'll have a look in few minutes, diner time.

**Comment by Ghalr** [comment]
Une partie des changements depuis Subsurface 4.6.2: --->Modifications apportées depuis Subsurface 4.6.2:

Ajout de la possibilité d'utiliser ctrl-molette de souris [...]  ---> Utilisation de Ctrl-molette de souris

Permet aux raccourcis claviers utilisant Ctrl/Cmd de fonctionner pour les versions traduites de Subsurface ---> Correction des raccourcis clavier Ctrl/Cmd dans les versions localisées de Subsurface

Corrige plusieurs inconsistances et fautes de frappe dans les textes de l'interface graphique -->
Amélioration de la cohérence du texte dans l'interface graphique

Assure le fonctionnement des titres de colonnes lorsque ces derniers sont traduits --> Correction de la traduction de la partie "Bloc de plongée"


I do assume that this changelog in english can be a bit improved.

== Massive improvements for several languages (including completing Italian, Greek)

10 languages at 100%, some are close to 100% it would be interesting to wait a bit to have them fully completed for at least 18 languages.

**Comment by dirkhh** [comment]
The logic of the ReleaseNotes is that they mention the notable changes... so yes, we have quite a few languages at 100%, but Greek and Italian came from <50% / <75% to fully translated. And Catalan was forgotten in the last release. Oops.
I have learned that "waiting for more translations to get to 100%" doesn't make sense - we already have two pull requests pending that I was holding back because they change strings...
And people may be traveling, busy with other stuff... I don't want to bother the volunteers too much, after all, they all, just like you, do this because it's fun and it helps a community - so it needs to stay fun.

Makes sense? 

So no, I wasn't ignoring the amazing work of the French translation team. That's very very appreciated.

File: _posts/announcing-subsurface-4-6-3.fr [hunk 1]
```diff
@@ -16,37 +16,37 @@ post_language: fr
...
-<li> Improve dive merging: deal with water temperature, better handling of dive sites</li>
-<li> Fix typo in saturation / desaturation rates used in decompression calculations; this has a small impact on some dive plan deco times</li>
-<li> Improve some icons used in Preferences dialog</li>
-<li> Fix CSV export of weights in lbs</li>
-<li> Various small fixes</li>
-<li> Massive improvements for several of the translations (including completing Italian and Greek)</li>
-<li> Update Russian user manual</li>
-<li> Add Catalan translation</li></ul>
+L'équipe de développement de Subsurface est fière de vous annoncer la version 4.6.3 de Subsurface, un carnet de plongée numérique et un planificateur de plongée open source pour Windows, Mac et Linux.
+
```
Comment (alexandrebelloni) [inline _posts/announcing-subsurface-4-6-3.fr]: Peut être pourrait on utiliser libre au lieu de open source.

File: _posts/announcing-subsurface-4-6-3.fr [hunk 2]
```diff
@@ -16,37 +16,37 @@ post_language: fr
...
+<strong>Linux:</strong>Une AppImage générique est disponible au téléchargement ici: <a href="https://subsurface-divelog.org/downloads/Subsurface-4.6.3-x86_64.AppImage">Subsurface-4.6.3-x86_64.AppImage</a>.  Téléchargez ce fichier, rendez le exécutable 
+<code>chmod +x Subsurface-4.6.3-x86_64.AppImage</code> et lancez simplement le fichier.
+<strong>Linux: Ubuntu: </strong>Ajoutez <code>ppa:subsurface/subsurface</code> a vos sources logiciel; le fichier .deb de ce PPA peut également être installé sur une version suffisamment récente de <strong>Debian</strong> et <strong>LinuxMint</strong>; voir la page de <a href="https://subsurface-divelog.org/download/">Téléchargement</a> ou utilisez <code>http://ppa.launchpad.net/subsurface/subsurface/ubuntu</code> pour obtenir la dernière version délivrée.
```
Comment (alexandrebelloni) [inline _posts/announcing-subsurface-4-6-3.fr]: vos sources logiciel -> vos sources logicielles

**Comment by dje29** [comment]
New change (amend) based on proposals from @Ghalr and @alexandrebelloni.
Thanks for the help

**Pull request merged**